.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
+/dist
```

Comment:
Added this to remove the noise around building the packages.
We can always build prior to publishing, without including the built files in the repo.

Reply:
Makes sense! Might be better to extract it into a separate PR to uncouple the changes?